### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by whimsicallyson**
Updates references to RStudio and RStudio Connect for the rebranding effort. 

It doesn't look like the README.md updated when I built, or knit? Not sure what I did wrong there.

There are still some links that will probably need to be updated before this goes live:
(none)

(updated link list, only base rstudio urls are migrating to posit.co immediately)

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright (c) 2021 RStudio, PBC
+Copyright (c) 2021 Posit Software, PBC
```
Comment (aronatkins) [inline LICENSE.md:3]: Does the date need to change with the company name? (same question applies to both LICENSE and LICENSE.md)

File: R/rsc_card.R [hunk 2]
```diff
@@ -53,7 +53,7 @@ rsc_card <- function(content) {
 #' @noRd
 widget_html.rsc_card <- function(id, style, class, ...) {
   htmltools::tagList(
-    # Necessary for RStudio viewer version < 1.2
+    # Necessary for Posit viewer version < 1.2
```
Comment (aronatkins) [inline R/rsc_card.R:56]: This is an RStudio IDE viewer version, I think. Same comment applies elsewhere.

**Review by aronatkins — APPROVED** [review]
A handful of RStudio IDE viewer references in comments should remain.

File: LICENSE.md (hunk 1 above)
Comment (whimsicallyson) [reply LICENSE.md]: yeah probably should, since Posit didn't exist in 2021...

**Comment by nkakade-rstudio** [comment]
Verified on local connect server: 
- Some instances of RStudio still found in paths and links. For e.g.
Screen Shot 2022-11-01 at 12 02 48 AM

**Comment by nkakade-rstudio** [comment]
Confirmed behavior with Allyson: 
things like package names are currently staying rstudio/whatever . in that repo, I don’t think there are any base [rstudio.com](http://rstudio.com/) links, which is all that is being migrated for launch. things like [docs.rstudio.com](http://docs.rstudio.com/) or any other subdomain is staying put for now

The changes in this PR are ready to be merged!

**Pull request merged**